PWGJE/Tasks/CMakeLists.txt

```diff
@@ -124,4 +124,8 @@ if(FastJet_FOUND)
                         SOURCES jetLundReclustering.cxx
                         PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::PWGJECore O2Physics::AnalysisCore FastJet::FastJet FastJet::Contrib
                         COMPONENT_NAME Analysis)
+    o2physics_add_dpl_workflow(jet-charm-hadronization
+                        SOURCES crCharmHadronization.cxx
+                        PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::PWGJECore O2Physics::AnalysisCore
```

Comment:
please change the name of your task to not include cr at the beggining. Something like hffragmentationfunction.cxx

Reply:
All right!